website/source/docs/agent/caching/index.html.md

```diff
@@ -232,3 +232,9 @@ vault {
   address = "http://127.0.0.1:8200"
 }
 ```
+
+## Learn
+
+Refer to the [Vault Agent
```

Comment:
In the other links you say "with Caching", here `with` is omitted.  It's fine either way, just letting you know it's inconsistent.

Reply:
yep, "Vault Agent Caching" (w/o with) is what I want :)